docker-compose.yml

```diff
@@ -209,7 +202,8 @@ services:
     image: scalableminds/webknossos-connect:master__190
     volumes:
       - "./conf/connect:/app/data"
-    network_mode: host
```

Comment:
This was necessary to make it work with `yarn start`. We could use an env var to overwrite it in the `proxy.js`.

Reply:
Seems to me this integration of wkconnect is turning into a complexity nightmare 🤦‍♂️ 
I think `yarn start` should not depend on docker at all. Submodules are also annoying. Forcing people to have the right Python version and deps seems also a bit too much. Perhaps we remove wkconnect from `yarn start`?

Reply:
I think for this integration it is perfectly fine to use docker, we all have it anyways. Why should `yarn start` not rely on docker?
We can either have the network overwritten by an env var or just do `docker run --rm -v … --network host scalableminds/webknossos-connect:…`. I think it makes sense to couple both closer without forcing one to install all the python deps.

Reply:
The problem is that Docker is incredibly resource-hungry on any non-Linux platform. When developing I wouldn't want to be forced to have it running. I see wkconnect more as an addon than a core component.

Reply:
Ok, I just removed wkconnect from the proxy.js.